apps/obsidian/src/components/NodeTagSuggestModal.tsx

```diff
@@ -0,0 +1,335 @@
+import { Editor } from "obsidian";
```

Comment:
Looks like we are mostly creating this from scratch?  Are there any existing obsidian components we can tie into without having to re-invent the wheel, cover all edge cases, etc?  List items, popovers, etc, etc.